clinica/utils/statistics.py

```diff
@@ -1,194 +1,6 @@
 # coding: utf8
 
 
-def permutation_test(vector_1, vector_2, number_of_permutation, tails=2):
```

Comment:
Where was this file used ?

Reply:
Archeology ;-) I removed this in 8da18fc:
> permutation_test, t_test, mann_whitney and fdr_correction_matrix have move to clinica_aramis. They were used by T. Jacquemont when he worked on connection-wise analysis and network-based statistics.

When the main code had moved to `clinica_aramis`, these functions were forgotten.